CIP-AFFIRM/README.md

```diff
@@ -0,0 +1,126 @@
+---
+CIP: ?
+Title: Affirmation Graph
+Category: Tools
+Status: Proposed
+Authors:
+  - Kieran Simkin <hi@clg.wtf>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/938
+    - https://forum.cardano.org/t/introducing-the-affirmation-graph-request-for-comment/138320
+Created: 2024-10-26
+License: CC-BY-4.0
+---
+
+## Abstract
+We need a truly decentralized alternative to KYC for when we need to establish the authenticity of a wallet or make some judgement about its trustworthiness in a systematic and repeatable manner. This CIP defines a mechanism whereby any member of the community can publicly vouch for the authenticity of a wallet by submitting an "affirmation" transaction, it also provides a mechanism to revoke such an affirmation, if, for example a wallet becomes compromised, or its owner turns out to be a charlatan. 
+It is consequently possible to observe the current UTxO set and draw an "Affirmation Graph", a multidigraph representing the trust relationships between wallets - we can then use an algorithm to assign trust rankings and decide whether or not to trust an unknown wallet.
+
+![Affirmation Graph example](graph.svg)
+
+## Motivation: why is this CIP necessary?
+When transacting with an unknown person, currently if we want to make sure we're not dealing with a bot or a grifter, we have to perform our own detective work - examining their current token holdings and transaction history, then we make some informed guess about what we're dealing with. This is both laborious and inconsistent. This CIP aims to provide a better alternative which is repeatable and automatable, this could drastically reduce the risk of fraud and accidental loss of funds.
+
+Now that we are in Cardano's Voltaire era; governance is top of the agenda, it would be really nice to be able to hold ballots where each community member gets an equal vote - true democracy requires it. Currently all voting is stake-weighted - ie. 1 Ada = 1 vote - this results in a plutocracy, not a democracy. In order to deliver a fair democracy, we must have some way of identifying unique individuals. Traditionally this would be done with KYC - each voting participant would be required to identify themselves with some form of government ID - however, this relies on a central authority having issued the ID, and another central authority verifying it. It also excludes people who wish to (or must) remain anonymous for whatever reason, as well as those who may not have any government ID, perhaps because they are stateless. 
+
+If we are to bank the unbanked, we should also provide ID for those who have none.
+
+In this CIP we propose a simple, truly decentralized mechanism for identity verification and trust.
```

Comment:
The claim of a "simple, truly decentralized mechanism for identity verification and trust" feels overstated. Based on the issues raised in my earlier comments, this CIP provides a mechanism for probabilistic trust relationships, which is not equivalent to robust identity verification. A more accurate phrasing might clarify that it offers a decentralized tool for assessing trust rather than verifying identity outright.

Reply:
It does also support robust identity verification in the form of affirmations by KYC providers. In fact it makes these KYC verifications portable so that you only need to do KYC once and it's portable to any dApp on Cardano.
That said, I take your point about overpromising, and will scale back some of the grand claims ;)